src/components/BookItem.js

```diff
@@ -1,49 +1,52 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faCircleNotch } from '@fortawesome/free-solid-svg-icons';
 import { deleteBook } from '../redux/books/booksSlice';
 
 function BookItem({ book }) {
   const dispatch = useDispatch();
   return (
-    <div className="d-flex book-info container">
+    <div className="d-flex book-info border-grey bg-white">
       <div className="book-detail w-50c">
         <div>
-          <div className="book-category">{book.category}</div>
-          <div className="book-title">{book.title}</div>
+          <div className="book-category txt-dark">{book.category}</div>
+          <div className="book-title txt-dark capitalize">{book.title}</div>
           <div className="author text-blue">{book.author}</div>
         </div>
         <div className="d-flex book-feedback w-50c">
-          <button type="button" className="text-blue btn-action line">
+          <button type="button" className="text-blue btn-action line pointer">
             Comment
           </button>
           <button
             onClick={() => {
               dispatch(deleteBook(book.item_id));
             }}
             type="button"
-            className="text-blue btn-action line pd-l10"
+            className="text-blue btn-action line pd-l10 pointer"
           >
             Remove
           </button>
-          <button type="button" className="text-blue btn-action line pd-l10">
+          <button
+            type="button"
+            className="text-blue btn-action line pd-l10 pointer"
+          >
             Edit
           </button>
         </div>
       </div>
       <div className="d-flex progress w-50c">
         <div className="d-flex w-50c progress-bar">
-          <FontAwesomeIcon icon={faCircleNotch} />
+          <div className="spin-container">
+            <div className="spin">{}</div>
+          </div>
           <div>
-            <div className="rating">74%</div>
-            <div>completed</div>
+            <div className="rating txt-dark">74%</div>
+            <div className="status">completed</div>
           </div>
         </div>
         <div className="w-50c progress-info">
           <div className="progress-stat">Current</div>
-          <div className="chapter">Chapter 18</div>
-          <button className="btn-progress" type="button">
+          <div className="chapter txt-dark">Chapter 18</div>
+          <button className="btn-progress bg-primary txt-upper" type="button">
             Update progress
           </button>
```

Comment:
- [ ] In the design the text on the button is on the same line, it is not on separate line as we can see i the screenshot below

![Screenshot from 2023-05-26 20-31-56](https://github.com/abiola-arobieke/book-store/assets/46388113/ffeb5f18-ed32-4a34-8a35-d0ad22566ec3)
![Screenshot from 2023-05-26 20-31-44](https://github.com/abiola-arobieke/book-store/assets/46388113/5665c4a5-b0cc-4863-80bb-ab4dc31ae05e)